src/devices_models/devices/renewable_generation.jl

```diff
@@ -203,13 +204,13 @@ function nodal_expression!(
         include_parameters(
             psi_container,
             ts_data_active,
-            UpdateRef{R}(ACTIVE_POWER, "get_rating"),
+            UpdateRef{R}(ACTIVE_POWER, "get_activepower"),
```

Comment:
undo this change. The full dispatch version is based on the model `P_re <= time_series_parameter*rating`